### DISCUSSION TITLE
p/b838d22a1702480c9d6df314f4a33e98/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#41] [Announcements]
# p/b838d22a1702480c9d6df314f4a33e98/

自从年前的寒假旅行回来之后，剩余的一周都一直宅在家里。导致了新年之后，即便开始继续上课了，状态也很糟，上午学校下课之后什么都不想做，很多时候只想看看同人小说、刷刷 youtube 之类的，甚至多邻国都漏了好几天。睡眠作息更是完全混乱了，通常不早于 2 点睡觉，这种状态非常糟糕。以前在工作的时候也有出现过，通常是感觉到生活不再有意义，所有的一切都是重复，典型的间歇性精神颓废。这周开始，这种状况开始好

https://blog.rxliuli.com/p/b838d22a1702480c9d6df314f4a33e98/

**Comment by xupt-chenxi** [comment]
吾辈总算更新了，本来这会准备过来问问最近怎么一直没更新来着，正好看到有更新

**Reply by rxliuli** [reply]
之前精神状态不好，没有发布的动力。。。

**Reply by xupt-chenxi** [reply]
在外面保护好自己，加油💪

**Comment by shenQQQ** [comment]
冬季抑郁

**Reply by rxliuli** [reply]
也许。。。
